src/labeler.ts

```diff
@@ -39,23 +43,33 @@ export async function run() {
       configPath
     );
 
-    const labels: string[] = [];
-    const labelsToRemove: string[] = [];
+    const labels: string[] = pullRequest.labels.map(label => label.name);
```

Comment:
May be it worth to create Set object at this point and avoid `!labels.includes(label)` check further

Reply:
I initially tried with a Set but the code was getting really ugly, I'll try that again 🤔

Reply:
Alright I pushed changes that use Sets instead, let me know what you think